Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -267,14 +306,20 @@ private void OnPreviewDragOver(object sender, DragEventArgs e)
             e.Handled = true;
         }
 
-        private void OnContextMenusForSettingsClick(object sender, RoutedEventArgs e)
+        private async void OnContextMenusForSettingsClick(object sender, RoutedEventArgs e)
         {
+            _viewModel.Hide();
+            await Task.Delay(100);
             App.API.OpenSettingDialog();
         }
 
 
-        private void OnDeactivated(object sender, EventArgs e)
+        private async void OnDeactivated(object sender, EventArgs e)
         {
+            _viewModel.Save();
```

Comment:
why are we calling save here everytime search window is not in focus/deactivated? that's too excessive. This save call does writes to disk.

Reply:
This is a mistake. It was put in the middle of the work for testing. I thought I deleted it.